.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -0,0 +1,84 @@
+name: Bug Report
+description: File a bug report
+title: ""
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Before you create the issue:
+        * Search for similar report among other reported issues.
+        * Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
+#  - type: input
+#    id: contact
+#    attributes:
+#      label: Contact Details
+#      description: How can we get in touch with you if we need more info?
+#      placeholder: ex. email@example.com
+#    validations:
+#      required: false
+  - type: dropdown
+    id: Mautic Version
+    attributes:
+      label: Mautic Version
+      description: What version of Mautic you are using? Please test to reproduce your bug with the [latest stable version of Mautic](https://www.mautic.org/mautic-releases) which might contain new fixes.  If you are able, please also check the latest development release.
+      options:
+        - 4.0.x series
+        - 3.3.3
+        - 3.3.x series
+        - 3.2.x series (not supported)
+        - 3.1.x series (not supported)
+        - 2.x series (not supported)
+    validations:
+      required: true
+  - type: input
+    id: PHP version
+    attributes:
+      label: PHP version
+      description: What PHP version are you using in your environment?
+      placeholder: ex. 7.3.27
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
+      - Firefox
+      - Chrome
+      - Safari
+      - Microsoft Edge
+      - Not relevant
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      value: "A bug happened!"
+    validations:
+      required: true
+   - type: textarea
+     id: what-happened
+     attributes:
+       label: How can we reproduce this issue?
+       description: Explain us carefuly how we can reproduce the bug you faced.
+       value: "1.
```

```suggestion
       value: |
          "1.
```

I think that for the multi-line we have to use the pipe, but I might be wrong here.  See how it's done in the [earlier section](https://github.com/mautic/mautic/pull/10258/files#diff-16caf9c5c96b17c5ca3f5762c8f05897e148bcfeb7723f42cff8b42f089d2f65R8) re. line spacing etc.